src/Social/Utils/StringFormatter.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Kanvas\Packages\Social\Utils;
+
+/**
+ * New event Manager to allow use to use fireToQueue.
+ */
+class StringFormatter
+{
+    /**
+     * Return an array of words that have hashtags
+     *
+     * @param string $text
+     * @return array
+     */
+    public static function getHashtagToString(string $text): array
+    {
+        preg_match_all('/#(\w+)/', $text, $hashTag);
+
+        return $hashTag[1];
```

Comment:
@arfenis dont we need to verify is array if not this will break?

Reply:
preg_match_all always returns an array of two positions even if both are empty.